### ISSUE TITLE
set app to listen on different context then root /

### ISSUE THREAD

**Issue by Jan-Marcek** [ldvm/LDVMi#60]
Hi there,

How can I easily run ldvmi on url path ```http://localhost:9000/ldvmi``` ? 

According https://www.playframework.com/documentation/1.2.7/configuration I should set ```http.path``` but it seems it is ignored. Could you help me?

I installed ldvmi from debian package.

Regards,
Jan

**Comment by jirihelmich** [comment]
Hi, even if you succeed setting this parameter (could you provide more details - are you adding it into config file, or passing as a JVM param?), I'm not sure that all parts of the app are ready to not to run on `/`.

Btw. relevant docs is at https://www.playframework.com/documentation/1.3.x/configuration (notice the version in the URL).

**Comment by Jan-Marcek** [comment]
Hi,
I added ```-Dhttp.path=/ldvmi/ ``` to ```/etc/default/ldvmi ```. 
I could see that it was used as JAVA_OPT paramater in ```ps ax | grep ldvmi```

**Comment by jirihelmich** [comment]
That seems to be fine. As this setting has no effect for WAR packages, I'm not sure it works for Debian packages. Try running the app manually first.

**Comment by jirihelmich** [comment]
No feedback, closing.

**Comment by Jan-Marcek** [comment]
hm, why there should be difference between non-debian version debian version? Debian package is just way how to put LDVMI in to your OS.  
Have you ever tried to run app in a different context?

**Comment by jirihelmich** [comment]
No, I haven't. There was no need for that, it's not designed while keeping
that in mind and it's not necessary for what we need.
Mon, 19 Oct 2015 v 12:29 odesílatel Jan Marcek <notifications@github.com>
napsal:

**Comment by jirihelmich** [comment]
As it seems that it's not just a nice to have feature for the ODN integration, reopening.
